### PR TITLE
Nightwatch asset failures

### PR DESCRIPTION
**Pull request by Matt-Butler**
### Summary
- All images are now stored in the repo. This will prevent random test failures.
- All tests now work offline

Thanks for contributing to Terra. 
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: packages/terra-demographics-banner/tests/jest/DemographicsBanner.test.jsx [hunk 1]
```diff
@@ -29,7 +28,7 @@ it('renders the banner that contains additional information', () => {
       dateOfBirth="May 9, 1993"
       gender="Male"
       identifiers={{ MRN: 12343, REA: '3JSDA' }}
-      photo={<img alt="My Cat" src="http://lorempixel.com/100/100/animals/7/" />}
+      photo={<img alt="My Cat" src="" />}
```
Comment (bjankord) [inline packages/terra-demographics-banner/tests/jest/DemographicsBanner.test.jsx]: Should this include the img src?
e.g. `My Cat`

File: packages/terra-demographics-banner/tests/jest/DemographicsBanner.test.jsx [hunk 2]
```diff
@@ -62,7 +61,7 @@ it('renders the banner appropriately for a person with gestational and post mens
       gender="Male"
       gestationalAge="April 5, 2016"
       identifiers={{ MRN: 12343, REA: '3JSDA' }}
-      photo={<img alt="My Cat" src="http://lorempixel.com/100/100/animals/7/" />}
+      photo={<img alt="My Cat" src="" />}
```
Comment (bjankord) [inline packages/terra-demographics-banner/tests/jest/DemographicsBanner.test.jsx]: Should this include the img src?
e.g. `My Cat`

File: packages/terra-demographics-banner/tests/jest/DemographicsBanner.test.jsx [hunk 3]
```diff
@@ -83,7 +82,7 @@ it('renders the banner properly for a deceased person with additional applicatio
       gender="Male"
       identifiers={{ MRN: 12343, REA: '3JSDA' }}
       personName="Johnathon Doe"
-      photo={<img alt="My Cat" src="http://lorempixel.com/100/100/animals/7/" />}
+      photo={<img alt="My Cat" src="" />}
```
Comment (bjankord) [inline packages/terra-demographics-banner/tests/jest/DemographicsBanner.test.jsx]: Should this include the img src?
e.g. `My Cat`

File: packages/terra-image/tests/jest/Image.test.jsx [hunk 4]
```diff
@@ -7,18 +7,18 @@ const imagecontainerStyle1 = {
   height: 200,
 };
```
Comment (bjankord) [inline packages/terra-image/tests/jest/Image.test.jsx]: Should these tests include a value for the `src` attribute?

File: packages/terra-demographics-banner/tests/jest/DemographicsBanner.test.jsx (hunk 1 above)
Comment (Matt-Butler) [inline packages/terra-demographics-banner/tests/jest/DemographicsBanner.test.jsx]: For the jest tests, I am just passing an empty string. If we add the image, then we need to mock it.

Comment (Matt-Butler) [inline packages/terra-demographics-banner/tests/jest/DemographicsBanner.test.jsx]: We could go ahead and mock the images like the approach here: https://facebook.github.io/jest/docs/webpack.html#handling-static-assets

Comment (bjankord) [inline packages/terra-demographics-banner/tests/jest/DemographicsBanner.test.jsx]: Cool, thanks for the clarification. Lets stick with the empty strings.

**Pull request merged**